### DISCUSSION TITLE
Figma Measure layers in prototypes

### DISCUSSION THREAD

**Discussion by bradryanbice** [ph1p/figma-measure#17] [Q&A] [answered]
I'd like a group of my Figma Measure layers to be visible in a prototype screen, to be used in documentation.

Is this possible somehow? Or at least the ability to toggle them on/off?

**Comment by bradryanbice** [comment, marked as answer]
Hmm, nevermind. Here's what happened:

Figma Measure-ments are put into a separate group outside of my prototype frames, so they were not being recognized. I dragged them in to one of my frames and now they show up fine.

**Reply by ph1p** [reply]
With the latest version it should run now anyway, because the group management works differently.
